.azure/pipelines/ci.yml

```diff
@@ -128,6 +128,9 @@ stages:
         steps:
         - powershell: ./eng/scripts/CodeCheck.ps1 -ci $(_InternalRuntimeDownloadArgs)
           displayName: Run eng/scripts/CodeCheck.ps1
+          env:
+            PULLREQUEST_SOURCEBRANCH: $(System.PullRequest.SourceBranch)
+            PULLREQUEST_TARGETBRANCH: $(System.PullRequest.TargetBranch)
```

Comment:
Recommend reverting change to this file. `$env:SYSTEM_PULLREQUEST_SOURCEBRANCH` and `$env:SYSTEM_PULLREQUEST_TARGETBRANCH` are already in the environment.
```suggestion
```